Dockerfile

```diff
@@ -27,6 +27,7 @@ FROM opensuse/leap:15.5
 # 4. RPM resolution logic
 # 5. Embedded artefact registry
 # 6. Network configuration
+# 7. RAW image modification on aarch64
```

Comment:
This isn't needed here. The intention was to explain the contents of this one big ass RUN line, but the aarch64 stuff is done at a different level after this.